### PR TITLE
tweak: visual improvement of OptionButton

### PR DESCRIPTION
**Pull request by Voomra**
## About the PR
OptionButton has an opaque background.  
OptionButton is no longer ugly! :tada: 

## Media
sample

## Changelog

:cl:
- tweak: visual improvement of OptionButton

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
frank-reynolds-danny-devito

File: Robust.Client/UserInterface/Controls/OptionButton.cs [hunk 1]
```diff
@@ -2,354 +2,363 @@
...
-            AddItem(label, id);
-        }
+            Children = { OptionsScroll }
+        };
+        panelContainer.PanelOverride = new StyleBoxFlat
+        {
+            BorderColor = new Color(53, 56, 76), //#35384c
+            BackgroundColor = new Color(36, 36, 43), //#24242b
+            BorderThickness = new Thickness(2, 2, 2, 2)
+        };
```
Comment (moonheart08) [inline Robust.Client/UserInterface/Controls/OptionButton.cs]: Hardcoded colors in engine are a *major* no. Please give the container a stylesheet class and add the actual panel stylebox to content.

Comment (Voomra) [reply Robust.Client/UserInterface/Controls/OptionButton.cs]: Done.
After that, i send PR to SS14, where add styling.

Comment (ShadowCommander) [reply Robust.Client/UserInterface/Controls/OptionButton.cs]: If you can, add it to this PR. The style file is StyleNano.cs

If you need help, feel free to ask.

Comment (moonheart08) [reply Robust.Client/UserInterface/Controls/OptionButton.cs]: > If you can, add it to this PR. The style file is StyleNano.cs
> 
> If you need help, feel free to ask.

StyleNano is in content, not here. Not possible.

**Pull request closed without merging**